actionlist/app/lib/commons.js

```diff
@@ -0,0 +1,22 @@
+/* eslint quotes: [2, "double"]*/
```

Comment:
This looks like a bad merge from `develop`. These eslint directives shouldn't be there and the `processInstanceJSON` and `preferenceJSON` should use single quotes (as on `develop`).